source/DataBase/DataBaseHandler.cpp

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <memory>
+#include <sstream>
+
+#include "DataBaseHandler.hpp"
+
+DataBaseHandler::DataBaseHandler(const std::string &dbName)
+    : database_(nullptr)
+{
+    int result = sqlite3_open(dbName.c_str(), &database_);
```

Comment:
After you change strings to chars you can use `DataBaseHandler::DataBaseHandler(const char* dbName)`
and just pass `dbName` to `sqlite3_open`.